moped-etl/arcgis/utils.py

```diff
@@ -116,6 +116,32 @@ def delete_features(feature_type):
         raise Exception(f"Delete features failed: {response_data}")
 
 
+def delete_features_by_project_ids(feature_type, project_ids):
+    """Deletes features from an arcgis online feature service associated with projects in a list ids.
+
+    Args:
+        feature_type (Str): the feature type we're adding: "points" or "lines"
+        project_ids (List): the project ids to delete
+
+    Raises:
+        Exception: if the deletion failes
+    """
+    endpoint = get_endpoint("deleteFeatures", feature_type)
+    joined_project_ids = ",".join(project_ids)
```

Comment:
i am running into a TypeError in this line, because `project_ids` is an array of ints. back in the main script you do a similar join to create `joined_project_ids`, and it kinda looks like you may have meant to pass that directly into this function. should be a quick fix either way 👍

Reply:
thanks for catching this detail 🙏